<launch><!--
    <arg name="ID" value="1" doc="The ID of the smallbot"/>
    <arg name="CLKPIN_BRUSH" value="37" doc="The Clock Pin of the sifter stepper motor"/>
    <arg name="DIRPIN_BRUSH" value="38" doc="The Direction Pin of the sifter stepper motor"/>
    <arg name="ENPIN_BRUSH" value="38" doc="The Enable Pin of the sifter stepper motor"/>

    <node name="Brush" pkg="small_bot" type="Brush.py">
        <param name="ID" value="$(arg ID)" type="str"/>
        <param name="CLKPIN_BRUSH" value="$(arg CLKPIN_BRUSH)" type="int"/>
        <param name="DIRPIN_BRUSH" value="$(arg DIRPIN_BRUSH)" type="int"/>
        <param name="ENPIN_BRUSH" value="$(arg ENPIN_BRUSH)" type="int"/>
    </node>
-->
</launch>
